metaflow/plugins/metadata/local.py

```diff
@@ -80,7 +80,9 @@ def register_task_id(
         except ValueError:
             self._new_task(run_id, step_name, task_id, attempt, tags, sys_tags)
         else:
-            self._register_code_package_metadata(run_id, step_name, task_id, attempt)
+            self._register_system_info_and_code_packaging_metadata(
```

Comment:
Minor nit - can we simply rename this to `_register_system_metadata`?

Reply:
Don't feel strongly... it is mainly for clarity.  We can use the shorter name - I can write a comment instead.